app/components/patient-summary.js

```diff
@@ -2,7 +2,26 @@ import Ember from 'ember';
 
 let PatientSummaryComponent = Ember.Component.extend({
   patient: null,
-  currentAssessment: "Stroke"
+  currentAssessment: "Stroke",
+
+  risksWithBirthdayStart: Ember.computed('patient.sortedRisks', 'patient.birthDate', 'currentAssessment', function(){
+    let birthRisk = this.get('patient.store').createRecord("risk-assessment", {date: this.get('patient.birthDate')});
+    let riskCode = this.get('patient.store').createRecord("codeable-concept", {text: this.get("currentAssessment")});
+    let rapc = this.get('patient.store').createRecord("risk-assessment-prediction-component", {probabilityDecimal: 0});
+    rapc.set('outcome', riskCode);
+    birthRisk.get('prediction').pushObject(rapc);
+    let risks = [birthRisk];
+    risks.pushObjects(this.get('patient.sortedRisks'));
+    return risks.filterBy('prediction.firstObject.outcome.text', this.get("currentAssessment"));
+  }),
+
+  assessments: Ember.computed('patient.risks', function() {
+    return this.get('patient.risksByOutcome').mapBy('key');
+  }),
+
+  changeAssessment: function() {
+    this.sendAction("action", this.get("currentAssessment"));
+  }.observes("currentAssessment")
 });
 
-export default PatientSummaryComponent;
\ No newline at end of file
+export default PatientSummaryComponent;
```

Comment:
For consistency, this export should be up at the top. Remove the `let` statement on line 3 and replace it with:

```js
export default Ember.Component.extend({
```